src/main/java/pb360/service/impl/GenerateRestApiServiceImpl.java

```diff
@@ -1,25 +1,34 @@
 package pb360.service.impl;
 
+import java.io.File;
+import java.io.IOException;
 import java.util.ArrayList;
 
 import java.util.Date;
 import java.util.List;
 
 import org.springframework.stereotype.Service;
 
+import com.fasterxml.jackson.core.JsonParser.Feature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import pb360.model.JsonDataModel;
 import pb360.model.MessageObject;
 import pb360.model.RestAPI;
 import pb360.service.GenerateRestApiService;
 
 @Service
 public final class GenerateRestApiServiceImpl implements GenerateRestApiService {
+	public String ControllerName;
+	public String link = "D:\\restapi\\JSON_Sample.json";
 
 	@Override
 	public MessageObject getRestApiData(String id) {
 		MessageObject getMessage = new MessageObject();
 		getMessage.setData("Get RestApi data successful: " + id);
 		getMessage.setType("GET");
 		getMessage.setDatetime(getCurrentDateTime().getDatetime());
+		ReadJson(link);
```

Comment:
please flow the rule of java function name

Reply:
i fixed it